pkg/storage/s3/s3.go

```diff
@@ -96,15 +99,18 @@ func (d *driver) createAndTagBucket(svc *s3.S3, clusterID string) error {
 		return err
 	}
 
+	d.Config.ManagedStorage = true
+
+	var tagSet []*s3.Tag
+	tagSet = append(tagSet, &s3.Tag{Key: aws.String("tectonicClusterID"), Value: aws.String(installConfig.ClusterID)})
+	for k, v := range installConfig.Platform.AWS.UserTags {
```

Comment:
We're unlikely to hit tgis code when `installConfig.Platform.AWS` is `nil`.  But it's probably tood to add a guard for it anyway, to avoid a panic if it does happen.

Reply:
+1

Reply:
updated in current push